### PR TITLE
javascript-loops-practice - Kyle Kettler

### PR DESCRIPTION
**Pull request by kyle-kettler**
SCR-20230511-hs6

### CODE REVIEW

File: javascript-loops-practice/filter-out-nulls.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const filteredValues = [];
+  for (let i = 0; i < values.length; i++) {
+    console.log(values[i]);
```
Comment (shawnkost) [inline javascript-loops-practice/filter-out-nulls.js:5]: `console.log` is only used for testing / debugging purposes, so we want to always remove these before submitting our exercises! Unless we specifically ask for you to include them in the instructions

File: javascript-loops-practice/sum-all.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported sumAll */
+function sumAll(numbers) {
+  let sum = 0;
+  for (let i = 0; i < numbers.length; i++) {
+    sum += numbers[i];
+    console.log(numbers[i]);
```
Comment (shawnkost) [inline javascript-loops-practice/sum-all.js:6]: Same note as above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks great Kyle, just a couple comments!

File: javascript-loops-practice/filter-out-nulls.js (hunk 1 above)
Comment (kyle-kettler) [reply javascript-loops-practice/filter-out-nulls.js:5]: Oh yeah, I was logging the loops to figure out what was happening and forgot they were in there. I'll remove and commit again!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**